client/components/mobile-phone-verification/mobile-phone-verification.test.js

```diff
@@ -239,7 +239,7 @@ describe("Mobile Phone Token verification: corner cases", () => {
     sessionStorage.clear();
   });
 
-  it("should not proceed if user is already active", async () => {
+  it("should not proceed if user is already verified", async () => {
```

Comment:
thanks for updating this kind of things, please double check comments and other parts like this, I will double check tomorrow.